src/ames_housing/io_managers/base_lakefs_io_manager.py

```diff
@@ -131,8 +136,8 @@ def handle_output(self, context: OutputContext, obj: Any) -> None:
                 context.log.debug(f"Writing file at: {self.get_path(context)}")
                 self.write_output(f, obj)
 
-            asset_without_repo_branch = "/".join(context.asset_key.path[2:])
-            commit = tx.commit(message=f"Add asset {asset_without_repo_branch}")
+            asset_name = "/".join(context.asset_key.path)
```

Comment:
Same as above

Reply:
Done in 320aeda385bd9c18086cce79ccc2f78e58c67650